student.rb

```diff
@@ -4,8 +4,8 @@
 class Student < Person
   attr_reader :classroom
 
-  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
-    super(age, name, parent_permission)
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission:)
     @classroom = classroom
```

Comment:
- [ ] Let's remove the colon at the end of parent_permission, it is rising an error. I would advise you to test the project before submitting it for review

Reply:
I don't know what's happening on your side, but its working perfectly fine on my side